### DISCUSSION TITLE
Is there a way to specify a hash algorithm for LDAP password reset?

### DISCUSSION THREAD

**Discussion by dougmeredith** [authelia/authelia#8266] [Q&A] [answered]
Password reset for LDAP uses an  SSHA hash, but I would like to configure it to use SSHA-512. I see that with the file backend it's possible to configure this. I've done a bit of searching, and can't find a way to do it for LDAP. Am I missing something?

**Comment by Crowley723** [comment, marked as answer]
Authelia doesn't handle the hashing of user passwords for the ldap backend. The password is sent to ldap where it is hashed according to the policies set in your ldap provider.

Depending on what ldap implementation you're using, that will change how you adjust the hashing algorithm you want used.

**Comment by dougmeredith** [comment]
I was completely unaware of this! Thank you for the information.

I'm using OpenLDAP 2.4.57 and I did some research on this and saw claims that setting olcDatabase={-1}frontend,cn=config olcPasswordHash to {SSHA512} would solve my problem. Unfortunately it has not, even after restarting the directory. The pasword is still hashed with SSHA If anyone has any tips on this, I'd be very grateful.
